app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessToilets.kt

```diff
@@ -28,6 +29,9 @@ class AddWheelchairAccessToilets : OsmFilterQuestType<WheelchairAccessToiletsAns
     override fun createForm() = AddWheelchairAccessToiletsForm()
 
     override fun applyAnswerTo(answer: WheelchairAccessToiletsAnswer, tags: Tags, geometry: ElementGeometry, timestampEdited: Long) {
-        answer.applyTo(tags)
+        when (answer) {
+            is WheelchairAccessToilets -> tags.updateWithCheckDate("wheelchair", answer.access.osmValue)
+            is NoToilet -> answer.applyTo(tags)
```

Comment:
We shouldn't ever reach this point as the NoToilet answer will not be shown for toilets, so is it best to do something else?

Reply:
Maybe throw instead?

```suggestion
            is NoToilet -> throw IllegalStateException()
```